portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,273 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+.background-block {
+    background-color: #e897c3 ;
+    margin-top: -34px;
+    padding-bottom: 400px;
+    width: 100%;
 }
 
-#greeting-container {
-  margin-top: 20px;
+.logo {
+    left: 15px;
+    position: absolute;
+    top: 105px;
 }
+
+.icon {
+    margin-top: 15%;
+}
+
+.bitmoji-headshot {
+    display: block;
+    margin: 30px auto;
+}
+
+.real-headshot {
+    margin-left: 200px;
+    margin-top: 100px;
+}
+
+h1 { 
+    font: normal normal 700 40px/42.4px 'Rockwell Extra Bold', 'Rockwell Bold', monospace; 
+    margin-top: 115px;
+    white-space: nowrap;
+}
+
+h2 { 
+    color: white;
+    font: normal normal 700 30px/32.4px 'Rockwell Extra Bold', 'Rockwell Bold';
+    padding-top: 115px;
+    white-space: nowrap;
+}
+
+h3 { 
+    font: normal normal 100 20px/21.4px Rockwell, 'Courier Bold', Courier, Georgia, Times, 'Times New Roman', serif;
+    margin-top: 30px;
+    white-space: nowrap;
+}
+
+h4 { 
+    color: black;
+    font: normal normal 100 25px/21.4px Rockwell, 'Courier Bold', Courier, Georgia, Times, 'Times New Roman', serif;
+    margin-top: 30px;
+    white-space: nowrap;
+}
+
+p.white {   
+    color: white;  
+    font: normal normal 400 20px/26px 'Rockwell Extra Bold', 'Rockwell Bold', monospace;
+}
+
+p.light-pink { 
+    color:#e897c3;
+    font: normal normal 400 20px/26px 'Rockwell Extra Bold', 'Rockwell Bold', monospace;
+}
+
+p.overview { 
+    color:gray;
+    font: normal normal 400 15px/21px 'Rockwell Extra Bold', 'Rockwell Bold', monospace;
+}
+
+p.field { 
+    color:#cf69a0;
+    font: normal normal 400 17px/23px 'Rockwell Extra Bold', 'Rockwell Bold', monospace;
+}
+
+.aboutme-block {
+    padding: 70px 20% 10px 10%;
+    text-align: center;
+}
+
+.center {
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+    text-align: center;
+}
+
+.text-center {
+    display: block;
+    padding: 0px 10% 9% 10%;
+    text-align: center;
+}
+
+.two-column-left {
+    float: left;
+    width: 50%;
+}
+
+.two-column-right {
+    float: left;
+    padding-left: -15%;
+    width: 40%;
+}
+
+.three-column-uniform {
+    float: left;
+    padding-top: 5%;
+    width: 33%;
+}
+
+.five-column {
+    float: left;
+    padding-top: 3%;
+    width: 20%;
+}
+
+.social-media:hover {
+    opacity: 0.5;
+    transition: 0.3s;
+}
+
+.projects:hover {
+    opacity: 0.6;
+    transition: 0.3s;
+    -webkit-tap-highlight-color: #e897c3;
+}
+
+.project-description {
+    visibility:hidden;
+}
+
+.projects {
+    border-radius: 15px;
+}
+
+.three-column-left {
+    background-color: white;
+    border-bottom-left-radius: 15px;
+    border-top-left-radius: 15px;
+    box-shadow: 0 0 1pt 0.5pt #d5d1d1;
```

Comment:
Omit leading “0”s in values. Check rest of the file for this.

https://google.github.io/styleguide/htmlcssguide.html#Leading_0s